docs/source/resources/frequently_asked_questions.ipynb

```diff
@@ -1196,7 +1196,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Finally, you might only want features that return a certain type. You can do this by using the `column_schema` attribute."
+    "Finally, you might only want features that return a certain type. You can do this by using the `column_schema` attribute. For more information on working with column schemas, take a look at [Transitioning from Variables to Woodwork](link)."
```

Comment:
we know the name of the notebook now, `transition_to_ft_v1.0.ipynb` so might as well put that here so that this doesn't need to be updated!